### ISSUE TITLE
What's the diff between smooth loss and loss?

### ISSUE THREAD

**Issue by WEIFZH** [sh0416/bpr#12]
Hi, there.
 Thank you for sharing your code, I am confused about smooth loss and loss you mentioned in train.py
could you give me more details about the differences between these two loss , it seems that smooth loss doesn't participate in backpropagation
```
optimizer.zero_grad()
loss = model(u, i, j)
loss.backward()
optimizer.step()
writer.add_scalar('train/loss', loss, idx)
smooth_loss = smooth_loss*0.99 + loss*0.01
```
thank you very much!

**Comment by sh0416** [comment]
Hi, @WEIFZH 
The smoothed loss is used for monitoring the training dynamics.
If we monitored the raw loss, then the loss curve is oscillated so much that it is confused whether the training worked or not.
You could report the raw loss and adjust the smoothing term through tensorboard.
Thanks.

**Comment by WEIFZH** [comment]
Got it! thanks for your kindly reply
